src/assets/mockData.json

```diff
@@ -0,0 +1,94 @@
+{
+  "projects":[
+    {
+      "id": "0",
+      "name": "Best Project",
```

Comment:
projectName

Reply:
actually name may be okay

Reply:
left as name, to keep consistency with client, blurb etc (think it's ok as project.name etc when accessing)